viewing_party/party.py

```diff
@@ -0,0 +1,117 @@
+def create_movie(movie_title, genre, rating):
+    new_movie = { }
+    if movie_title==None or genre==None or rating==None:
+        return None
+    new_movie['title'] = movie_title
+    new_movie['genre'] = genre
+    new_movie['rating'] = rating
+    return new_movie
+
+def add_to_watched(user_data, movie):
+    watched_list = user_data["watched"]
+    watched_list.append(movie)
+    return user_data
+
+def add_to_watchlist(user_data, movie):
+    watch_list = user_data["watchlist"]
+    watch_list.append(movie)
+    return user_data
+
+def watch_movie(user_data, title):
+    watch_list = user_data["watchlist"]
+    watched_list = user_data["watched"]
+    for movie_dict in watch_list:
+        if title==movie_dict["title"]:
+            watched_list.append(movie_dict)
+            watch_list.remove(movie_dict)
```

Comment:
even though remove works great here, be careful using it in a for loop. It can cause side effects. Here is a link that explains it more with examples https://thispointer.com/python-remove-elements-from-a-list-while-iterating/